### ISSUE TITLE
Allow changing the scale of UI

### ISSUE THREAD

**Issue by Yura52** [zed-industries/zed#7820]
### Check for existing issues

- [X] Completed

### Describe the feature

The default UI scale is too small to me, and it seems that there is no way to increase it. For comparison:
- In Sublime, there is the "ui_scale" setting. Would be amazing to see this feature in Zed!
- In VSCode, it it possible to configure the global zoom level which scales everything (UI, fonts, etc.).

**Comment by jackTabsCode** [comment]
The `ui_font_size` setting makes everything bigger in the editor from text, to icons, to buttons. Perhaps try that?

**Comment by dzakarov** [comment]
The icons are still too small even with font size 32.

**Comment by dzakarov** [comment]
Currently two settings affect the scaling:
```
"ui_font_size": 32,
"buffer_font_size": 32,
```
I would expect the third one:
```
"ui_icon_size": 48

```

**Comment by brekker23** [comment]
I agree there should be a separate option for the ui_icon_size, I have the text set to a size I like but the icons are too small, it would make sense to be able to adjust them independently

**Comment by d1y** [comment]
Just my personal opinion: this will only make the interface chaotic
You should use `ui_font_size` to resize the interface and interface icons 😳

**Comment by Quicksilver151** [comment]
i have the opposite problem. the ui is too large. changing font doesnt change the size of the top pannel. did they end up hardcoding the ui scale to fit extra large monitor resolutions? in addition, all font sizes display text at double the true size of the font on screen.

**Comment by niallobrien** [comment]
> i have the opposite problem. the ui is too large. changing font doesnt change the size of the top pannel. did they end up hardcoding the ui scale to fit extra large monitor resolutions? in addition, all font sizes display text at double the true size of the font on screen.

Same issue here, especially on a laptop with a smaller screen

**Comment by Quicksilver151** [comment]
> Same issue here, especially on a laptop with a smaller screen

@niallobrien 

for whatever reason it looks like they made the decision to use scale factor of 2 by default.

create this environment variable and set it to 1 then launch zed

`GPUI_X11_SCALE_FACTOR=1 zed`
